v0/destinations/indicative/transform.js

```diff
@@ -0,0 +1,63 @@
+const { EventType } = require("../../../constants");
```

Comment:
does indicative has id resolution thing? did we check with the user flow from unidentified tracking to identified tracking

Reply:
Yes. Updated identify logic to include alias call to map the userId correctly